Query was executed successfully

Query was executed successfully

[(10),(11),(12)]

[("A"),("C")]

[An error expected at this place for the query "redimension_store(A,C)". And it failed with error code = scidb::SCIDB_SE_STORAGE::SCIDB_LE_CHUNK_OUT_OF_BOUNDARIES. Expected error code = scidb::SCIDB_SE_STORAGE::SCIDB_LE_CHUNK_OUT_OF_BOUNDARIES.]

[("A"),("C")]

Query was executed successfully

Query was executed successfully

[]

